### PR TITLE
netpol proxy for kube-burner

### PR DESCRIPTION
**Pull request by venkataanil**
## Type of change

- [x] New feature

## Description
kube-burner uses the pod created with this image to communicate with workload pods

## Related Tickets & Documents
https://github.com/kube-burner/kube-burner/pull/679
https://github.com/cloud-bulldozer/images/pull/12

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -3,7 +3,7 @@ ENGINE ?= podman
 ORG ?= cloud-bulldozer
 REGISTRY ?= quay.io
 REG = $(REGISTRY)/$(ORG)
-REPOS = perfapp  etcd-perf nginx frr netpol-scraper nginxecho eipvalidator sampleapp
+REPOS = perfapp  etcd-perf nginx frr netpol-scraper nginxecho eipvalidator sampleapp netpolproxy
```
[the remark this answers is not in the document]
Comment (jtaleric) [reply Makefile:6]: done

File: netpolproxy/netpolproxy.go [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+	resultsMutex.Unlock()
+}
+
+// Get results from all pods
+func getResults(cts map[string][]connection) {
+	semaphore := make(chan struct{}, parallelConnections)
+	for pod, _ := range cts {
+		semaphore <- struct{}{}
+                resWg.Add(1)
+                go getPodResult(pod, semaphore)
```
Comment (rsevilla87) [inline netpolproxy/netpolproxy.go:203]: Indentation of this golang file doesn't look correct, can you run gofmt and push again?

Comment (venkataanil) [reply netpolproxy/netpolproxy.go:203]: sure Raul, thanks

**Review by jtaleric — APPROVED** [review]
/lgtm - my only hangup is the default to `:9002` not an issue for our work

**Pull request merged**